src/cli/tc.rs

```diff
@@ -18,12 +18,24 @@ use crate::print_schema;
 
 pub fn run() -> Result<()> {
   let cli = Cli::parse();
+  // set the log level
+  let long_env_filter_var_name = "TAILCALL_LOG_LEVEL";
+  let short_env_filter_var_name = "TC_LOG_LEVEL";
+
+  // Check if TAILCALL_LOG_LEVEL is defined, otherwise use TC_LOG_LEVEL
+  let filter_env_name = if env::var(long_env_filter_var_name).is_ok() {
+    long_env_filter_var_name
+  } else {
+    short_env_filter_var_name
```

Comment:
We should use the operators that are available on Result and Option here to make this concise.